### PR TITLE
Edits to Home Pages

### PR DESCRIPTION
**Pull request by lesleymatheson**

### CODE REVIEW

File: templates/index.html [hunk 1]
```diff
@@ -94,10 +94,9 @@ <h3 class="ud-feature-title">Tune Privacy Settings</h3>
             <i class="bi-file-bar-graph"></i>
           </div>
           <div class="ud-feature-content">
-            <h3 class="ud-feature-title">Generate Anonymous Datasets</h3>
+            <h3 class="ud-feature-title">Differentially Private Queries</h3>
             <p class="ud-feature-desc">
-              Datasets that can be monetized with minimal risk to individuals
-              in the original data.
+              SMP enables differentially private aggregation queries.
```
Comment (chaukap) [inline templates/index.html:99]: *SDP

File: templates/index.html [hunk 2]
```diff
@@ -357,7 +361,10 @@ <h2>Any Questions? Answered</h2>
             </button>
             <div id="collapseSeven" class="accordion-collapse collapse">
               <div class="ud-faq-body">
-                Yes!
+                The CCPA now has an enforcement arm, the CRPA.  The regulations can
+                be found here: https://cppa.ca.gov/regulations/.  Differential Privacy techniques
```
Comment (chaukap) [inline templates/index.html:365]: Let's make this a hyperlink 
```
<a href="https://cppa.ca.gov/regulations/" target="_blank">The regulations can be found here</a>.
```

File: templates/index.html [hunk 3]
```diff
@@ -485,9 +505,14 @@ <h2>Any Questions? Answered</h2>
...
-                <br>
-                Additional query capability such as VARIANCE, STANDARD DEVIATION, MEDIAN.
+                The product roadmap for SDP includes user tracking dashboards for data owners
+                to monitor activity in the form of dashboards and the ability for owners to set
+                privacy budgets for individual users.  These improvements will increase an owner's
+                ability to detect attacks. SDP envisions additional query capabilities
+                such as VARIANCE, STANDARD DEVIATION, MEDIAN.  Additional advanced query mechanisms
+                are envisiioned which include clamping, advanced average algorithms,
+                and customizable Exponential scoring functionality.  For greater detail check out
+                our product roadmap page HERE.
```
Comment (chaukap) [inline templates/index.html:515]:
```
<a href="/product_roadmap">here</a>
```

**Review by chaukap — CHANGES_REQUESTED** [review]

File: templates/index.html (hunk 1 above)
Comment (lesleymatheson) [reply templates/index.html:99]: SDP sorry about that.

File: templates/index.html (hunk 2 above)
Comment (lesleymatheson) [reply templates/index.html:365]: Great!

File: templates/index.html (hunk 3 above)
Comment (lesleymatheson) [reply templates/index.html:515]: awesome

**Pull request merged**